### DISCUSSION TITLE
Security & "Quicksupport" like functionality?

### DISCUSSION THREAD

**Discussion by Entelin** [rustdesk/rustdesk#948] [Q&A]
I have the rustdesk server working with preshared key on docker. My primary use case is to provide support to end users, and so I have put my custom named (including the key) windows exe as a link on my website to make it easy for users to connect. However my understanding is that those users technically could then just use my server with the full client feature-set, including giving the link to others, and remoting into each others systems. Or of course just random people finding the link on my site and using it. I don't want this, I want full functionality for myself, but end users should have the ability to read me their ID, but that's basically it, similar to teamviewer quicksupport. Is there a way to lock the server down in this way?

Thanks

**Comment by rustdesk** [comment]
Short term, no
Long term, yes.

**Reply by RedTiger26** [reply]
Will "long term" mean months, years or decades from now?
